.github/workflows/master.yml

```diff
@@ -69,3 +69,22 @@ jobs:
           GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
           package_root: "packages/core"
           tag_prefix: "v"
+
+      # Publish techdocs-core to PyPI
+      - uses: actions/checkout@master
+      - name: Set up Python ${{ matrix.python-version }}
```

Comment:
I think this could br a second master build workflow that only runs on changes to techdocs right? Also a bit strange to check out master again here imo

Reply:
@Rugvip Make sense, thanks! 👍  Can you take a look at the changes to see if it looks correct to you?